cockroachdb/values.yaml

```diff
@@ -362,6 +362,9 @@ init:
   labels:
     app.kubernetes.io/component: init
 
+  # Additional annotations to apply to this Job.
+  jobAnnotations: {}
+
```

Comment:
At the top of this file it says 
```
# Generated file, DO NOT EDIT. Source: build/templates/values.yaml
```
;)
If you don't update it, these changes will be overwritten.

Reply:
Done. ✅